Marlin/Marlin_main.cpp

```diff
@@ -4562,6 +4562,13 @@ inline void gcode_M105() {
 
 #endif
 
+  #if DISABLED(MIN_COOLING_SLOPE_DEG)
```

Comment:
We have to use `#ifndef` for settings that take values. So, once it's patched this up I'll merge this asap!